### DISCUSSION TITLE
Is there a way to have a "password profile" (i.e. central password) for use in SSH sessions?

### DISCUSSION THREAD

**Discussion by AustinHarisch** [Eugeny/tabby#9669] [Q&A]
I SSH to switches all the time, and I have to configure my password for each and every switch that I connect to. That password is managed by a TACACS+ server, which means if I change my password, I'd have to update it for every switch I have an entry for. Is there a way to have a "password profile" that I can apply to every SSH connection so I'd only have to update it in one spot? i.e. I'd create a new profile with my TACACS+ username and password and call it "TACACS Credentials" or something like that. Or, if that isn't possible, is there at least a way for me to edit the config in bulk so I can change all of the passwords at once?

**Comment by noahrod** [comment]
I'm looking for something similar, did you find a way?

**Comment by Clem-Fern** [comment]
You can edit session options at group level. Profiles in Tacacs group should inherit the credentials you defined for the group.

![image](https://github.com/user-attachments/assets/67cbe6f6-0576-42f6-8649-15d0973badb5)
![image](https://github.com/user-attachments/assets/896b5c9a-5942-44f8-918d-c99396b5f6bc)

**Reply by ianaflous** [reply]
> You can edit session options at group level. Profiles in Tacacs group should inherit the credentials you defined for the group.

It's not working. I suggested this #10076. It's not using groups specifically, but it will prevent saving the same credentials in many profiles. It has been merged, it's available in the [Latest release](https://github.com/Eugeny/tabby/releases/latest).
